CardData.py

```diff
@@ -5,16 +5,20 @@
 IMAGE_SIZES = ['original', 'high', 'medium', 'low', 'thumbnail']
 
 CRAFT_VALUES = {}
-CRAFT_VALUES['Common'] = {"standard": 30, "premium": 200, "upgrade": 100}
-CRAFT_VALUES['Rare'] = {"standard": 80, "premium": 400, "upgrade": 200}
-CRAFT_VALUES['Epic'] = {"standard": 200, "premium": 800, "upgrade": 300}
-CRAFT_VALUES['Legendary'] = {"standard": 800, "premium": 1600, "upgrade": 400}
+CRAFT_VALUES[1] = {"standard": 30, "premium": 200, "upgrade": 100}
```

Comment:
should be `CRAFT_VALUES[RARITIES.COMMON]` (or similar) (also below)
Otherwise defeats the purpose of Enums

Reply:
1 is now the value that represents 'Common' in the game files, 2 -> 'Rare' etc. I will abstract this away behind an enum.

Reply:
`1, 2, 4, 8` for rarity, aren't binary values a strange choice ? It's not like rarities could be cumulated.